modules/fx/arrow-fx/src/test/kotlin/arrow/fx/QueueTest.kt

```diff
@@ -214,7 +163,149 @@ class QueueTest : UnitSpec() {
       }
     }
 
-    tests("BoundedQueue", queue =
+    fun boundedStrategyTests(
+      ctx: CoroutineContext = Dispatchers.Default,
+      queue: (Int) -> IO<Queue<ForIO, Int>>
+    ) {
```

Comment:
You can call `allStrategyTests` within `boundedStrategyTests` and that will ensure that those test run for `boundedStrategyTests` to pass. That also reduces calls sites.